aggregator/capture/aggregator.go

```diff
@@ -87,6 +89,14 @@ func (agg *aggregator) AddUntimed(
 	return nil
 }
 
+// TODO(xichen): implement this.
+func (agg *aggregator) AddForwarded(
+	metric aggregated.Metric,
+	metadata metadata.ForwardMetadata,
+) error {
+	return nil
```

Comment:
nit: would it be better to return a not implemented error just to be safe?

Reply:
Sure can do, I'll add the implementation in the next PR but doesn't hurt either.